### PR TITLE
feat: add the packages submitted by infoengine1337

### LINKED ISSUE #402: 【パッケージの更新】アイテムずらし等
### 名前
infoengine1337

### データ
``` xml
<packages>
	<package>
		<id>hebiiro/BuildStairs</id>
		<name>BuildStairs</name>
		<overview>タイムラインのアイテムをずらしたり整列したりします</overview>
		<description>タイムライン上のアイテムを一定フレーム感覚でずらしたり、一括で整列したりできます。</description>
		<developer>hebiiro</developer>
		<dependencies>
			<dependency>aviutl1.10</dependency>
			<dependency>exedit0.92</dependency>
		</dependencies>
		<pageURL>https://github.com/hebiiro/AviUtl-Plugin-BuildStairsWindow#readme</pageURL>
		<downloadURL>https://github.com/hebiiro/AviUtl-Plugin-BuildStairs/releases/latest</downloadURL>
		<latestVersion>2.1.0</latestVersion>
		<files>
			<file directory="true">plugins/BuildStairs</file>
			<file>plugins/BuildStairs.auf</file>
		</files>
		<releases>
			<release version="2.1.0">
				<archiveIntegrity>sha384-2TjkrJw5PEtt7z0ph8P3UXqWC3TUQS8JtaZ9A23m/vmFpqM/+77a2JAPDxieIuOZ</archiveIntegrity>
				<integrities>
					<integrity target="plugins/BuildStairs.auf">sha384-RjjM4H9Zxr9n4chL7KZUOElpcTxJuLLzAh3oZsKZ/svW+dzxrKB07peYrQI9J44x</integrity>
				</integrities>
			</release>
		</releases>
	</package>
	<package>
		<id>hebiiro/SplitText</id>
		<name>SplitText</name>
		<overview>テキストオブジェクトをバラバラに分解します</overview>
		<description>Aviutl拡張編集上のテキストオブジェクトを一文字ごとに別オブジェクトとして新たに生成します。
</description>
		<developer>hebiiro</developer>
		<dependencies>
			<dependency>aviutl1.10</dependency>
			<dependency>exedit0.92</dependency>
		</dependencies>
		<pageURL>https://github.com/hebiiro/AviUtl-Plugin-SplitText#readme</pageURL>
		<downloadURL>https://github.com/hebiiro/AviUtl-Plugin-SplitText/releases/latest</downloadURL>
		<latestVersion>1.7.0</latestVersion>
		<files>
			<file directory="true">plugins/SplitText</file>
			<file>plugins/SplitText.auf</file>
		</files>
		<releases>
			<release version="1.7.0">
				<archiveIntegrity>sha384-R5uqW0sHLT+pePJhXbM5IDUFCB9UNPvUYV2M2TUr+AysMYyRZEcU5R8ye+CqveYy</archiveIntegrity>
				<integrities>
					<integrity target="plugins/SplitText.auf">sha384-yokIL3bAf4lXtA0DqBLdh61ErbxOXOqwYOT4HljXBmn+MiQTrZR0uv5qsIcKrOhX</integrity>
				</integrities>
			</release>
		</releases>
	</package>
	<package>
		<id>hebiiro/SelectScene</id>
		<name>SelectScene</name>
		<overview>シーンを簡単に切り替えられるようにします</overview>
		<description>Aviutl拡張編集内部のシーンを簡単に切り替えられるウィジェットを表示します。</description>
		<developer>hebiiro</developer>
		<pageURL>https://github.com/hebiiro/AviUtl-Plugin-SelectScene#readme</pageURL>
		<downloadURL>https://github.com/hebiiro/AviUtl-Plugin-SelectScene/releases/latest</downloadURL>
		<latestVersion>2.0.0</latestVersion>
		<files>
			<file>plugins/SelectScene.auf</file>
			<file directory="true">plugins/SelectScene</file>
		</files>
		<releases>
			<release version="2.0.0">
				<archiveIntegrity>sha384-MGBJTDi58ph04aXW1Z27Ed2o8/j0udcVwwiAm55Qpr20D43L9AERzbPjInO9y+Ob</archiveIntegrity>
				<integrities>
					<integrity target="plugins/SelectScene.auf">sha384-W/ZqdKzatgw+fD73PWep63ClXMN/+O2n38EoxcpHByx7tAFocYx2thoMi8xxD9bS</integrity>
				</integrities>
			</release>
		</releases>
	</package>
	<package>
		<id>hebiiro/SelectEasing</id>
		<name>SelectEasing</name>
		<overview>イージング番号を画像から選択できるようにします。</overview>
		<description>イージング番号の選択時にイージング番号の表示と、easeモードの場合はベジエ曲線の表示を行います。
ease.dllの利用を推奨します。</description>
		<developer>hebiiro</developer>
		<dependencies>
			<dependency>aviutl1.10</dependency>
			<dependency>exedit0.92</dependency>
		</dependencies>
		<pageURL>https://github.com/hebiiro/AviUtl-Plugin-SelectEasing#readme</pageURL>
		<downloadURL>https://github.com/hebiiro/AviUtl-Plugin-SelectEasing/releases/latest</downloadURL>
		<latestVersion>4.5.0</latestVersion>
		<files>
			<file directory="true">plugins/SelectEasing</file>
			<file>plugins/SelectEasing.auf</file>
		</files>
		<releases>
			<release version="4.5.0">
				<archiveIntegrity>sha384-rktAcO6vo2Oki55xX5qIuYthzcdZFpXz1xkLCa7mejKSbT/k0wLtTBFUoVwBNW82</archiveIntegrity>
				<integrities>
					<integrity target="plugins/SelectEasing.auf">sha384-+N3+N2foDURoTvcpwaKy9847iu0A2pqF491amO/p54pD3uk3Ok6+uda3sX7Iys7g</integrity>
				</integrities>
			</release>
		</releases>
	</package>
	<package>
		<id>ePi/ease</id>
		<name>ease</name>
		<overview>よりスマートに書き直されたイージングスクリプト</overview>
		<description>undofish版イージングをより扱いやすくすることを目的としたスクリプト。
カスタムオブジェクトのベジエ曲線を配置せず、数値指定でグラフを表現することが可能である。</description>
		<developer>ePi</developer>
		<dependencies>
			<dependency>aviutl1.10</dependency>
			<dependency>exedit0.92</dependency>
		</dependencies>
		<pageURL>https://scrapbox.io/ePi5131/ease</pageURL>
		<downloadURL>https://scrapbox.io/ePi5131/ease</downloadURL>
		<latestVersion>1.0</latestVersion>
		<files>
			<file>ease.dll</file>
			<file>script/ease_common.lua</file>
			<file>script/@ease.tra</file>
		</files>
		<releases>
			<release version="1.0">
				<archiveIntegrity>sha384-6h3mWXA+B8uVw6WfePzDUleCDV6yTwtziSASH0n3xByI1i/CkRbKGVxOA6ROfv1C</archiveIntegrity>
				<integrities>
					<integrity target="ease.dll">sha384-928LaRvK77QLMFPc5nRCxhzc/oilvmmLcLoYIQg9J6lGJINN0S8C99Jt7+RBY/0F</integrity>
					<integrity target="script/ease_common.lua">sha384-MLIPqtTkqNHlyz8LWFF9+IwVbdNkXiEv41NicZrIdwbrUnx43iya/PBHMRLqXazX</integrity>
					<integrity target="script/@ease.tra">sha384-rHpTS4Admu7jOfM0Jucs16hY30S8jZdB/Y5/s6+qdjjD0CVMl8fP9I849nmrjute</integrity>
				</integrities>
			</release>
		</releases>
	</package>
</packages>
```

### タイムスタンプ
2022/11/09 17:38:03

### LINKED ISSUE #403: 【パッケージの更新】スクリプト並び替えパッチ
### 名前
infoengine1337 scriptsortpatch

### データ
``` xml
<packages>
	<package>
		<id>hebiiro/ScriptSortPatch</id>
		<name>script_sort_patch</name>
		<overview>黒窓プラグインとスクリプト並び替えプラグインの不具合と競合を解消します</overview>
		<description>黒窓プラグインとスクリプト並び替えプラグインの不具合と競合を解消します。</description>
		<developer>hebiiro</developer>
		<dependencies>
			<dependency>aviutl1.10</dependency>
			<dependency>exedit0.92</dependency>
			<dependency>hebiiro/DarkenWindow</dependency>
			<dependency>rikky/scriptSort1|rikky/scriptSort2</dependency>
		</dependencies>
		<pageURL>https://github.com/hebiiro/AviUtl-Plugin-script_sort_patch#readme</pageURL>
		<downloadURL>https://github.com/hebiiro/AviUtl-Plugin-script_sort_patch/releases/latest</downloadURL>
		<latestVersion>2.0.0</latestVersion>
		<files>
			<file>plugins/script_sort_patch.auf</file>
		</files>
		<releases>
			<release version="2.0.0">
				<archiveIntegrity>sha384-lpBZGjLfnl/SonKmu7+Rebu36FYdNR++g4ii257nsfTqYfbkIlMGwY+5/rCtVOOY</archiveIntegrity>
				<integrities>
					<integrity target="plugins/script_sort_patch.auf">sha384-2+4BAqQBLPPo6KBC5vyFtZAJ6DWpeymp0FkuBayW1jA0YjeVMku/2S/AbACwRzPE</integrity>
				</integrities>
			</release>
		</releases>
	</package>
</packages>
```

### コメント
過去バージョンについては記述せず

### タイムスタンプ
2022/11/09 17:43:12

### LINKED ISSUE #404: 【パッケージの更新】イージング設定時短プラグイン
### 名前
infoengine1337 easingquicksetup

### データ
``` xml
<packages>
	<package>
		<id>kumrnm/EasingQuickSetup</id>
		<name>EasingQuickSetup</name>
		<overview>AviUtl 拡張編集のイージング設定を補助するプラグイン</overview>
		<description>パラメーター付きのイージングが選択された場合、パラメーター設定ダイアログをその場で表示します</description>
		<developer>蝙蝠の目</developer>
		<dependencies>
			<dependency>aviutl1.10</dependency>
			<dependency>exedit0.92</dependency>
		</dependencies>
		<pageURL>https://github.com/kumrnm/aviutl-easing-quick-setup#readme</pageURL>
		<downloadURL>https://github.com/kumrnm/aviutl-easing-quick-setup/releases/latest</downloadURL>
		<latestVersion>1.1.0</latestVersion>
		<files>
			<file>plugins/easing_quick_setup.auf</file>
		</files>
		<releases>
			<release version="1.1.0">
				<archiveIntegrity>sha384-MSAF2biby803lPT3xKariRh0bqVo+JD3COG1eJMbWUKeM4cpQVy5lsTyq/wfWdZO</archiveIntegrity>
				<integrities>
					<integrity target="plugins/easing_quick_setup.auf">sha384-MSAF2biby803lPT3xKariRh0bqVo+JD3COG1eJMbWUKeM4cpQVy5lsTyq/wfWdZO</integrity>
				</integrities>
			</release>
		</releases>
	</package>
</packages>
```

### コメント
過去バージョンについては記述なし

### タイムスタンプ
2022/11/09 17:50:39

### LINKED ISSUE #405: 【パッケージの更新】真・グループ制御
### 名前
infoengine1337  authentic grouping

### データ
``` xml
<packages>
	<package>
		<id>kumrnm/AuthenticGrouping</id>
		<name>AuthenticGrouping</name>
		<overview>AviUtl 拡張編集のグループ制御の機能を改善するプラグイン</overview>
		<description>グループ制御を入れ子にした際の親グループの制御範囲を変更し、グループ制御が複数のグループ制御を制御できるようにする。</description>
		<developer>蝙蝠の目</developer>
		<dependencies>
			<dependency>aviutl1.10</dependency>
			<dependency>exedit0.92</dependency>
		</dependencies>
		<pageURL>https://github.com/kumrnm/aviutl-advanced-mouse-operation#readme</pageURL>
		<downloadURL>https://github.com/kumrnm/aviutl-advanced-mouse-operation/releases/latest</downloadURL>
		<latestVersion>1.1.0</latestVersion>
		<files>
			<file>plugins/AdvancedMouseOperation.auf</file>
		</files>
		<releases>
			<release version="1.1.0">
				<archiveIntegrity>sha384-Ef9jJjYNpuY5Jkz6SoeOjz6QCthuNd/R82pY+7wEi3czpHm7g4ut3uTgH7TueR1d</archiveIntegrity>
				<integrities>
					<integrity target="plugins/AdvancedMouseOperation.auf">sha384-Ef9jJjYNpuY5Jkz6SoeOjz6QCthuNd/R82pY+7wEi3czpHm7g4ut3uTgH7TueR1d</integrity>
				</integrities>
			</release>
		</releases>
	</package>
</packages>
```

### コメント
過去バージョンについては記述なし

### タイムスタンプ
2022/11/09 17:53:44

### LINKED ISSUE #406: 【パッケージの更新】マウスオペレーション改
### 名前
infoenigne1337 マウスオペレーション改

### データ
``` xml
<packages>
	<package>
		<id>kumrnm/AdvancedMouseOperation</id>
		<name>マウスオペレーション改</name>
		<overview>AviUtl拡張編集メインウィンドウ上での編集操作を強化するプラグイン</overview>
		<description>メインウィンドウ上での編集操作としてマウスコマンドを追加</description>
		<developer>蝙蝠の目</developer>
		<dependencies>
			<dependency>aviutl1.10</dependency>
			<dependency>exedit0.92</dependency>
		</dependencies>
		<pageURL>https://github.com/kumrnm/aviutl-advanced-mouse-operation#readme</pageURL>
		<downloadURL>https://github.com/kumrnm/aviutl-advanced-mouse-operation/releases/latest</downloadURL>
		<latestVersion>1.0.0</latestVersion>
		<files>
			<file>plugins/AdvancedMouseOperation.auf</file>
		</files>
		<releases>
			<release version="1.0.0">
				<archiveIntegrity>sha384-Ef9jJjYNpuY5Jkz6SoeOjz6QCthuNd/R82pY+7wEi3czpHm7g4ut3uTgH7TueR1d</archiveIntegrity>
				<integrities>
					<integrity target="plugins/AdvancedMouseOperation.auf">sha384-Ef9jJjYNpuY5Jkz6SoeOjz6QCthuNd/R82pY+7wEi3czpHm7g4ut3uTgH7TueR1d</integrity>
				</integrities>
			</release>
		</releases>
	</package>
</packages>
```

### タイムスタンプ
2022/11/09 18:01:58

### LINKED ISSUE #407: 【パッケージの更新】カメラ補助プラグイン
### 名前
infoengine1337 CameraAssist

### データ
``` xml
<packages>
	<package>
		<id>nazono/CameraAssist</id>
		<name>カメラ補助プラグイン(要ランタイム・省メモリ)</name>
		<overview>Aviutlカメラの移動方法を変更し、より使いやすくするプラグイン</overview>
		<description>Aviutlカメラの移動方法を変更し、より使いやすくするプラグイン</description>
		<developer>謎の生物</developer>
		<dependencies>
			<dependency>aviutl1.10</dependency>
			<dependency>exedit0.92</dependency>
		</dependencies>
		<pageURL>https://www.nicovideo.jp/watch/sm39701988</pageURL>
		<downloadURL>https://drive.google.com/u/0/uc?id=1Hyh1f6kpblxwI7-eESj-Kxd_FU4Abpb0&amp;export=download</downloadURL>
		<latestVersion>1.7</latestVersion>
	</package>
</packages>
```

### タイムスタンプ
2022/11/09 18:11:31

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Add the packages submitted by Infoengine1337.

Tasks
- [x] Relax dependent AviUtl versions where possible.

## Related Issue

- closes #402 
- closes #403 
- closes #404 
- closes #405 
- closes #406 
- closes #407

## Checklist:

- [x] Updated the modification date in `mod.xml`.

### CODE REVIEW

**Review by hal-shu-sato — COMMENTED** [review]
All packages can be installed correctly.
But I have one point to suggest.

File: v3/packages.json [hunk 1]
```diff
@@ -4010,6 +4282,52 @@
...
     },
+    {
+      "id": "nazono/CameraAssist",
+      "name": "カメラ補助プラグイン(要ランタイム・省メモリ)",
+      "overview": "Aviutlカメラの移動方法を変更し、より使いやすくするプラグイン",
+      "description": "Aviutlカメラの移動方法を変更し、より使いやすくするプラグイン",
+      "developer": "謎の生物",
+      "pageURL": "https://www.nicovideo.jp/watch/sm39701988",
+      "downloadURLs": [
+        "https://drive.google.com/u/0/uc?id=1Hyh1f6kpblxwI7-eESj-Kxd_FU4Abpb0&export=download"
```
[the remark this answers is not in the document]
Comment (mitosagi) [reply v3/packages.json]: It seems to be robust to version changes.

**Comment by mitosagi** [comment]
I have split the CameraAssist plugin with and without Runtime.

**Review by hal-shu-sato — APPROVED** [review]
It's good!

**Pull request merged**